addon/-private/dsl.js

```diff
@@ -1,8 +1,10 @@
 import { as } from './properties/as';
+import { blurrable } from './properties/blurrable';
```

Comment:
I think you forgot to include these files in the PR
* addon/-private/properties/blurrable.js
* addon/-private/properties/focusable.js

Reply:
That would explain it 😅 updated!

Reply:
nevermind! I've just found them.